optimum_energy_rubocop.yml

```diff
@@ -73,6 +88,39 @@ Layout/SpaceBeforeFirstArg:
 Layout/SpaceInsideStringInterpolation:
   EnforcedStyle: space
 
+###########
+# METRICS #
+###########
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Enabled: true
+  Max: 100
+  Exclude:
+    - 'spec/**/*'
+
+Metrics/ClassLength:
+  Enabled: true
+  Max: 300
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
```

Comment:
This one seems useful too. Why was it disabled?

Reply:
The problem is it's not discerning. It doens't handle DSLs well, and there are rare exceptions when we want longer methods. I think it's better to call this out in PR reviews.